Docker/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM php:7.4-cli
+
+RUN apt-get update && apt-get install -y --no-install-recommends git && apt-get purge -y --auto-remove
```

Comment:
I would suggest dropping this Dockerfile, using `php:8-cli` directly instead in the Makefile, and take advantage of git on the client

Reply:
Ah sorry, that is not possible because you use git inside Docker

Reply:
But maybe consider `php:8-cli-alpine` then, which is much smaller and with a much lighter build

Reply:
Example https://github.com/FreshRSS/FreshRSS/blob/896f9d33637ec510937bc13a34f98f669795ab49/Docker/Dockerfile-Alpine#L1-L9

Reply:
Done!